sqlobject/src/main/java/org/jdbi/v3/sqlobject/SqlObjectConfigurer.java

```diff
@@ -13,11 +13,15 @@
  */
 package org.jdbi.v3.sqlobject;
 
-import org.jdbi.v3.extension.ExtensionConfig;
-
-public class SqlObject implements ExtensionConfig<SqlObject> {
-    @Override
-    public SqlObject createCopy() {
-        return new SqlObject();
-    }
+/**
+ * Configures a {@link SqlObjectConfig}.
+ */
+@FunctionalInterface
+public interface SqlObjectConfigurer {
```

Comment:
Is this useful as a standalone interface, or would `Consumer<SqlObjectConfig>` work as well?